src/js/behaviors/sortable-list.js

```diff
@@ -6,13 +6,36 @@ export default Behavior.extend({
     handle: '.js-sort',
     item: '.js-draggable',
   },
+  initialize() {
+    this.shouldDisable = this.getOption('shouldDisable');
+  },
+  /* istanbul ignore next: sensible but unused default */
+  shouldDisable() {
+    return this.view.collection.length < 2;
+  },
+  collectionEvents: {
+    'update change:id': 'updateDisabled',
+  },
+  updateDisabled() {
+    const shouldDisabled = this.shouldDisable();
+    this.view.$el.toggleClass('is-dragable', !shouldDisabled);
```

Comment:
Not a huge deal since the classname in the scss is the same, but typo.`is-draggable`